### DISCUSSION TITLE
IBEX on FFPE Samples

### DISCUSSION THREAD

**Discussion by LJY1207** [IBEXImagingCommunity/ibex_imaging_knowledge_base#69] [Q&A]
Hello!

I've been doing IBEX on precut 4 micron FFPE tissue sections from pathology recently and have noticed that the sections sometimes adhere well throughout cycles and sometimes begin to detach and float away before I even begin staining (post dewax and antigen retrieval). It also doesn't seem to be tissue specific, as I've tested a couple different tissues types and have noticed it happening occasionally in both. I'm wondering if anyone else has experienced this, and if so, is it something that can be attributed to tissue quality or section thickness? If anyone has solutions or ideas on how to address this issue, I would love to hear it. 

Thank you!!

**Comment by radtkea** [comment]
Hello!

Thanks for your question!

I am sorry that you are having difficulty with tissue adherence. Very frustrating!

Indeed, tissue adherence has been shown to be impacted by section thickness and tissue type!

1. [Gambella and colleagues](https://link.springer.com/article/10.1007/s00418-017-1558-4) did an exhaustive comparison of many variables including tissue thickness, slide age, tissue age, etc. Based on their recommendations (and my experience), I would continue to cut thin tissue sections (3-4 micron). I would also advise you to use Superfrost slides. [These](https://us.vwr.com/store/product/4646061/null) are my favorite. 
2. Others have shown that certain tissues are less adherent than others, e.g., fatty tissues, necrotic tissues. In our experience, the skin and lung can be tricky. [Eng and colleagues](https://www.nature.com/articles/s42003-022-03368-y#MOESM2) performed a very exhaustive comparison of tissue loss with [CyCIF](https://www.nature.com/articles/ncomms9390), a robust cyclic imaging method employing H2O2 for dye inactivation. They observed more tissue loss with normal (surprising for me) than malignant tissues. Please see Figure 2e and Supplementary Figure 3 in the Eng paper.

I would recommend the following:
1. Cut thin (3-4 micron tissue sections).
2. Use Superfrost slides
3. BAKE your tissues for 30 minutes at 60C before performing dewaxing and antigen retrieval. More details [here](https://vectorlabs.com/products/blog/how-to-prevent-tissue-and-reagent-loss-on-your-slides.html). 
4. Wash samples gently. Automate if possible with a Leica Bond or other [device](https://biogenex.com/nanovip-100/?v=7516fd43adaa).

For fixed frozen sections, we always recommend using [Chrome Alum Gelatin](https://www.newcomersupply.com/product/chrome-alum-gelatin-adhesive/#:~:text=Newcomer%20Supply%20Chrome%20Alum%2DGelatin,tissue%20sections%20and%20microscopic%20slides.) coated slides. We have shown that this is not so critical for FFPE tissue sections. However, the adhesive works very well!

We tried these adhesives. Chrome Alum Gelatin is superior!
1. Poly-lysine
2. Haupt Gelatin Adhesive
3. Tissue Capture pen
5. Aminoalkylsilane Slide Adhesive
6. BIOBOND Tissue Section Adhesive

Best of luck! We've expanded the number of FFPE conjugates for IBEX. Feel free to ask here for an antibody recommendation if you don't see your target in the knowledge-base yet.

Have a great one,
A

**Reply by LJY1207** [reply]
Thank you so much! This is immensely helpful. Very grateful for the time and effort you put into this, and I will give your recommendations a try!

All the best,
Y

**Comment by radtkea** [comment]
Aww, thanks!

Happy I could be helpful. You're very welcome!

Have a lovely day,
A

**Comment by radtkea** [comment]
One more thing. The baking temperature for tissue adherence is highly variable.

The Leica Bond does 60C 30 minutes.
https://acdbio.com/sites/default/files/MK%2051-151%20TN%20LS%20RED%20with%20ICW.pdf

Other reports list 56C and caution against above 60C to prevent epitope loss.
https://vectorlabs.com/products/blog/how-to-prevent-tissue-and-reagent-loss-on-your-slides.html

Others list for longer at 56C:
https://www.ihcworld.com/_technical_tips/section_falloff_tips.htm

Here's one that says 62C:
https://www.labce.com/spg605391_proper_slide_drying_for_paraffin_sectioned_slides.aspx

I think 60C for 30 minutes will work. 

Best wishes,
A
